dist/index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Exquisite Kitchen</title>
+    <link
+      rel="stylesheet"
+      href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.2/css/bootstrap.min.css"
+      integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z"
+      crossorigin="anonymous"
+    />
+    <link rel="stylesheet" href="main.css" />
+  </head>
+  <body>
+    <div class="container">
+      <ul class="nav nav-pills nav-fill">
+        <li class="nav-item">
+          <a class="nav-link" href="index.html">
+            <img src="imgs/exquisite.png" class="logo" alt="main logo" />
+          </a>
+        </li>
+        <li class="nav-item pt-4">
+          <a class="nav-link active" href="#">Home</a>
+        </li>
+        <li class="nav-item pt-4">
+          <a class="nav-link" href="#">Menu</a>
+        </li>
+        <li class="nav-item pt-4">
+          <a class="nav-link" href="#">Contact</a>
+        </li>
+      </ul>
+    </div>
```

Comment:
- [ ] You are expected to make index.html very lean. Pleas remove this nav element and dynamically create it with JavaScript instead.